perllib/Open311/UpdatesBase.pm

```diff
@@ -177,6 +179,18 @@ sub _process_update {
         $email_text = $request->{email_text};
     };
 
+    if ($request->{latest_data_only}) {
```

Comment:
I believe this should mention `extras` otherwise it'll never match?

```suggestion
    if ($request->{extras}{latest_data_only}) {
```

Reply:
Tests certainly would be good, then, wouldn't they!